etc/helm/pachyderm/values.yaml

```diff
@@ -38,7 +38,8 @@ dash:
   # podLabels specifies labels to add to the dash pod.
```

Comment:
Seems like perhaps a formatter got run on this file?

Reply:
> Also note #6524 has shipped, so dash references are now `console`

ah right, I'll pull in those changes

Reply:
oof yes, will revert that